### ISSUE TITLE
Some kind of baseline

### ISSUE THREAD

**Issue by szepeviktor** [codespell-project/codespell#2704]
Codespell is a great software!

I have 26 false positives in my repo like `neeed ==> need`, "neeed" is a company name.
If I ignore a word it will be ignored _everywhere_.

Is there a way to ignore a word in one file only?

**Comment by matlupi** [comment]
yes, create a file (e.g. `.codespell.ignore`) with the following content (one word per line)

```
neeed
```

and call `codespell` as follows

```bash
codespell --ignore-words .codespell.ignore
```

This should now be solved.

**Comment by szepeviktor** [comment]
Thank you.

**Comment by szepeviktor** [comment]
I know, I know. I use `typos`.
